packages/analytics-types/src/diagnostic-event.ts

```diff
@@ -0,0 +1,10 @@
+export interface DiagnosticEvent {
```

Comment:
We might want to make this internal to the DiagnosticProvider. Since it won't be immediately used by customers, it might be confusing compared to "real" events.

Reply:
Put it `packages/analytics-core/src/diagnostic.ts` without export.